news/_posts/2017-10-27-lummen-for-life.markdow

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "Lummen for Life 1-12-2017"
+date:   2017-10-27 20:00:00
+description: Lummen for Life 1 December 2017.
+permalink: /news/lummen-for-life-1-12-2017/
+---
+Zin om een Lummens goed doel te steunen en een leuke avond te beleven? Op vrijdag 1 December 2017 organiseert Basket Lummen een toernooi ter voordele van **Music for Life**, het toernooi zal doorgaan van 20u - 22u in [Sporthal Vijfsprong](/club/sporthal/).
+
+## Toernooi
+
+Het formaat van dit toernooi zal volgens de ['3 tegen 3'](http://www.basketbal.vlaanderen/competitie/3x3) regels verlopen. 
+Er wordt gespeeld op hoge ringen, vandaar dat je minstens 12 jaar moet zijn om deel te kunnen nemen.
+
+## Inschrijven
+
+Inschrijven kan vanaf 12 jaar en kost 3 euro per wedstrijd per ploeg. Als je wil inschrijven kan dit door een email te sturen naar [linda.doggen@telenet.be](mailto://linda.doggen@telenet.be).
+
+## Goede doel
+
+De inkomsten van dit toernooi komen ten goede aan vzw **Avali**. Zwemclub Avali biedt sinds 1997 zwemlessen en –trainingen aan voor personen met een verstandelijke beperking. Meer informatie vind je op [www.avali.be](https://www.avali.be/).
```

Comment:
Misschien nog het logo van Music for Life (vlammetje) en van Avali zelf erbij te zetten?

Reply:
@marliesvlaeyen logo's zijn toegevoegd